assets/src/application/ui/input/Button/style.scss

```diff
@@ -18,7 +18,7 @@
 	box-shadow: 0 0 0 1px var(--ee-color-primary-low-contrast);
 }
 
-.ee-btn-base.ee-btn-base {
+.ee-btn-base {
```

Comment:
uh oh.. is this going to cause issues elsewhere?
do you remember what you added this in to fix?

Reply:
yes, I have added that recently because there was mismatch between these 2 buttons:

![image](https://user-images.githubusercontent.com/1477453/82355031-56830900-9a0a-11ea-8de9-2acaea216c38.png)

Reply:
![image](https://user-images.githubusercontent.com/1477453/82355058-5d118080-9a0a-11ea-98da-628c3e1120c1.png)

Reply:
but then added a helper css class name and that specificity wasn't needed, but had to research the impact of that specificity

Reply:
was added here: https://github.com/eventespresso/event-espresso-core/pull/2849/files#r427355084